pull_request_template.md

```diff
@@ -8,6 +8,6 @@ What this PR includes
 ----
 Brief description of what was done to meet the PR goal.
 
-How to test / verify these changes
+How this change was tested
 ----
-This is an optional section! Only include it if applicable.
\ No newline at end of file
+Describe testing done on the change, or why this is not applicable.
```

Comment:
How is "How this change was tested" different from "Describe testing done on the change"?

Reply:
Section header, followed by instructions for that section.  Is this what you mean?

Reply:
Ignore my comment.  I looked at the diff completely wrong. I thought it was adding a new section.